src/CMakeLists.txt

```diff
@@ -6,3 +6,36 @@ add_library(sqlpp-mysql
 		detail/connection_handle.cpp)
 
 install(TARGETS sqlpp-mysql DESTINATION lib)
+install(TARGETS sqlpp-mysql EXPORT sqlpp11-connector-mysqlTargets)
+include(CMakePackageConfigHelpers)
+
+write_basic_package_version_file(
+    "${CMAKE_CURRENT_BINARY_DIR}/cmake/sqlpp11-connector-mysqlConfigVersion.cmake"
+        VERSION 0.29
```

Comment:
This has to be automated. Otherwise it will be outdated whenever the next release comes :-)

Reply:
Sorry was a quick hack. Used the PROJECT_VERSION variable instead now, cause I can't think of anything better.

Reply:
Also ran cmake-format cause there was inconsistencies in the code that I copied from multiple places.